### PR TITLE
Add CakePHP version directly in the toolbar

### PR DESCRIPTION
**Pull request by antograssiot**
qisn7gbfew

sorry the gif quality is pretty bad but don't want to spend time on this 😢

### CODE REVIEW

File: webroot/css/toolbar.css [hunk 1]
```diff
@@ -130,6 +131,13 @@ p {
 	color: #ab47bc;
 }
 
+#cake-version {
```
Comment (antograssiot) [inline webroot/css/toolbar.css]: There are some of those already bot.
Mark let me know if you want me to transform this into a class. no problem

Comment (ADmad) [inline webroot/css/toolbar.css]: @antograssiot You can fix all the ID selectors :)

**Comment by inoas** [comment]
Don't worry, we have to change all those icon/references to that icon to svg at some point anyway.

**Comment by antograssiot** [comment]
@inoas I don't understand your comment.... I don't see the link between the cake version display that I added and icon/svg stuff ?!

**Comment by inoas** [comment]
Ah you meant your animated gif, not the cakephp icon, nvm sorry for the noise.

**Comment by markstory** [comment]
Seems reasonable. I was wondering if there would be value in having a 'packages' panel that listed out the versions of all the packages an application used?

**Comment by antograssiot** [comment]
So the idea would be to parse the composer.lock ?

**Comment by markstory** [comment]
@antograssiot I assume there is something we could use in composer, or by reading the lock file.

**Comment by antograssiot** [comment]
@markstory I've quickly worked on your idea and the result could look like the following.
The tooltip of each row would show the package's description and by clicking on the package name it open the packagist page of the package in a new tab (I found that the homepage info is not reliable but we can also use this or both with minor work if you prefer).

packagist

Is it near what you had in mind? Should I push those changes and do I need to remove the Cakephp version from the toolbar then ?

**Comment by lorenzo** [comment]
@antograssiot I love it! It would be really nice if it could tell you if there is a new version you can install, like showing outdated packages

**Comment by josegonzalez** [comment]
Probably worth putting the packages list in a separate pr so we can get this one merged?

**Comment by antograssiot** [comment]
Except if this one doesn't make sense to others @savant

**Comment by dereuromark** [comment]
People can always disable it locally if they dont want it, right?

**Comment by antograssiot** [comment]
I don't see how @dereuromark but I might have missed something

**Comment by dereuromark** [comment]
I thought it worked like this to disable a tab:
```php
Configure::write(
    'DebugKit.panels',
    [
        foobar => false
    ]
);
```

**Comment by antograssiot** [comment]
Bit it is not a real panel... it is not part of the config

**Comment by dereuromark** [comment]
OK, a new PR is best to not block this one from moving forward while discussing the new feature.

**Comment by markstory** [comment]
@antograssiot That is *AMAZING*

**Pull request merged**